src/lib/upload_to_s3.js

```diff
@@ -46,6 +47,9 @@ export default async function uploadToS3 (
     }
     debug(`wrote source file to ${tmp.path} for ${artifactName}`);
 
+    const stat = fs.statSync(tmp.path);
```

Comment:
if we're in an async function already, have we considered importing `mz/fs`?  For this case, it's probably not worth the concern, but it might be a good thing to have since we already have the overhead of an async function.